### PR TITLE
Address for the other PEI, NB, NL, ON

### PR DESCRIPTION
**Pull request by elynch303**
this should provide address for all provincial gov except MB as i need to wait for the site to update form the election yesterday.

@jpmckinney local tests look good let me know if you need anything changed for this to get merged

### CODE REVIEW

File: ca_nb/people.py [hunk 1]
```diff
@@ -43,8 +46,11 @@ def scrape(self):
                 primary_org="legislature", name=name, district=district, role="MLA", party=party, image=photo_url
             )
             if phone:
-                p.add_contact("voice", phone, "legislature")
+                p.add_contact("voice", phone, "constituency")
```
Comment (elynch303) [inline ca_nb/people.py:49]: i noticed that this was incorrectly labeled

File: ca_nl/people.py [hunk 2]
```diff
@@ -37,15 +37,12 @@ def scrape(self):
...
-                name=" ".join(reversed(member["name"].split(","))).strip(),
-                district=member["district"]
-                .replace("&apos;", "'")
-                .replace(
-                    " - ",
-                    "\u2014",
-                ),  # match messy boundary data
+                name=name,
+                district=district,
                 role="MHA",
```
Comment (elynch303) [inline ca_nl/people.py:46]: need name and district for address data so made them vars

**Comment by jpmckinney** [comment]
Please `pip install pre-commit` and `pre-commit install`. I added a GitHub Actions workflow now to report linting issues, but you can avoid them via pre-commit.

File: ca_ns/people.py [hunk 3]
```diff
@@ -53,4 +46,37 @@ def scrape(self):
...
+                        address = civic_address_alt
+                    else:
+                        address = civic_address
+                        address.remove(address[0])  # remove civic address
+                else:
+                    if len(business_address) > 0:
+                        address = business_address
+
+            address = list(map(str.strip, address))
+            p.add_contact("address", "\n".join(address), "constituency")
```
Comment (elynch303) [inline ca_ns/people.py:81]: helps to handle a few more missing address and deals with bad inconsistent dom

**Pull request merged**